tests/releases/reference_docs_test.go

```diff
@@ -41,10 +41,10 @@ func TestReferenceDoc(t *testing.T) {
 				Version: version.Name,
 				Kind:    crd.Spec.Names.Kind,
 			}
-			if gvk.Group == "core.cnrm.cloud.google.com" {
+			if strings.HasSuffix(gvk.Group, "core.cnrm.cloud.google.com") {
 				continue
 			}
-			if gvk.Group == "customize.core.cnrm.cloud.google.com" {
+			if strings.HasSuffix(gvk.Group, "customize.core.cnrm.cloud.google.com") {
```

Comment:
Having done the "core.cnrm.cloud.google.com" suffix check above, this should now be redundant.